recipes/git-pull-request/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "git-pull-request" %}
+{% set version = "5.0.0" %}
```

Comment:
```suggestion
{% set version = "6.0.1" %}
```

Reply:
I actually wanted to start with 5.0.0 and make the others "automagically" with the bot, but OK